### PR TITLE
requests package updated

### PR DESCRIPTION
**Pull request by adeelehsan**
requests package upgraded from 2.9.1 to 2.22.0 and execute make upgrade executed.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -25,16 +25,16 @@ upgrade: ## update the pip requirements files to use the latest releases satisfy
...
-	pip-compile --upgrade -o requirements.txt requirements/base.in
-	pip-compile --upgrade -o playbooks/roles/aws/templates/requirements.txt.j2 requirements/aws.in
-	pip-compile --upgrade -o requirements3.txt requirements/ses-limits.in
-	pip-compile --upgrade -o util/elasticsearch/requirements.txt requirements/elasticsearch.in
-	pip-compile --upgrade -o util/jenkins/update_celery_monitoring/requirements.txt requirements/celery.in
-	pip-compile --upgrade -o util/jenkins/check_celery_progress/requirements.txt requirements/celery_progress.in
-	pip-compile --upgrade -o util/jenkins/requirements-cloudflare.txt requirements/cloudflare.in
-	pip-compile --upgrade -o util/pingdom/requirements.txt requirements/pingdom.in
-	pip-compile --upgrade -o util/vpc-tools/requirements.txt requirements/vpc-tools.in
+	pip-compile --upgrade --rebuild -o requirements/pip-tools.txt requirements/pip-tools.in
```
Comment (feanil) [inline Makefile]: What was the issue being caused by the cache?  Why add this option? Can you add a comment to that effect.

**Review by feanil — COMMENTED** [review]
One question otherwise looks good.

File: Makefile (hunk 1 above)
Comment (adeelehsan) [reply Makefile]: Cache did't have any issues. I have removed it
@feanil can you please have look again.

**Comment by feanil** [comment]
Looks like you're failing all the python 3 builds, you'll have to make sure the requirements update works for both python2 and python3.

**Comment by feanil** [comment]
at the very least, you need to update wsgi-ref to work with python3. Or add back the `; python_version == "2.7"` in the `.in` or `constraints.txt` file

**Comment by adeelehsan** [comment]
@feanil can you please have a look again? All the tests are passed

**Comment by adeelehsan** [comment]
@feanil can you please review this PR?

**Review by feanil — APPROVED** [review]

**Pull request merged**